### PR TITLE
[Qol] Add an option to import settings

### PR DESCRIPTION
**Pull request by prateau**
## What are the changes?
Added an option in the manage data menu to import and export settings

## Why am I doing these changes?
I oftenly clear the cookies of my browser and so I often have to re-enter all my settings manually. This is not fun.

Maybe this could be extended in the future to key mapping settings

## What did change?
I added two entries in the manage data menu to import and export the settings.

As I'm native in french but not in the other languages, I only added translation for english and french. Let me know if I need to translate the menu entries in the other language (with google translate)

### Screenshots/Videos
https://github.com/pagefaultgames/pokerogue/assets/56865723/9d36bc7f-d1f8-44d3-80ff-c3fd19c2e61b

## How to test the changes?
This can be tested by exporting and importing settings with the new option menu.

## Checklist
- [x] There is no overlap with another PR?
- [x] The PR is self-contained and cannot be split into smaller PRs?
- [x] Have I provided a clear explanation of the changes?
- [x] Have I tested the changes (manually)?
    - [x] Are all unit tests still passing? (`npm run test`)
- [x] Are the changes visual?
  - [x] Have I provided screenshots/videos of the changes?

### CODE REVIEW

File: src/locales/fr/menu-ui-handler.ts [hunk 1]
```diff
@@ -18,6 +18,8 @@ export const menuUiHandler: SimpleTranslationEntries = {
   "exportSlotSelect": "Sélectionnez l’emplacement depuis lequel exporter les données.",
   "importData": "Importer données",
   "exportData": "Exporter données",
+  "importSettings": "Importer paramètres",
+  "exportSettings": "Exporter paramètres",
```
Comment (Adri1) [inline src/locales/fr/menu-ui-handler.ts:22]: Don't forget to also add these lines to all other languages local files
You can put them in English, tranalsation teams will do thier work once you added them

**Review by Adri1 — CHANGES_REQUESTED** [review]

File: src/locales/fr/menu-ui-handler.ts (hunk 1 above)
Comment (prateau) [reply src/locales/fr/menu-ui-handler.ts:22]: Ok I didn't knew this. It's done

**Review by Adri1 — DISMISSED** [review]

File: src/locales/pt_BR/menu-ui-handler.ts [hunk 2]
```diff
@@ -18,6 +18,8 @@ export const menuUiHandler: SimpleTranslationEntries = {
   "exportSlotSelect": "Selecione um slot para exportar.",
   "importData": "Importar dados",
   "exportData": "Exportar dados",
+  "importSettings": "Import Settings",
+  "exportSettings": "Export Settings",
```
Comment (josericardo-fo) [inline src/locales/pt_BR/menu-ui-handler.ts:22]:
```suggestion
  "importSettings": "Importar configurações",
  "exportSettings": "Exportar configurações",
```

File: src/locales/de/menu-ui-handler.ts [hunk 3]
```diff
@@ -18,6 +18,8 @@ export const menuUiHandler: SimpleTranslationEntries = {
   "exportSlotSelect": "Wähle einen Slot zum Exportieren.",
   "importData": "Daten importieren",
   "exportData": "Daten exportieren",
+  "importSettings": "Import Settings",
+  "exportSettings": "Export Settings",
```
Comment (CodeTappert) [inline src/locales/de/menu-ui-handler.ts:22]:
```suggestion
  "importSettings": "Einstellungen importieren",
  "exportSettings": "Einstellungen exportieren",
```

**Review by Arxxer — COMMENTED** [review]
ES localization done

File: src/locales/zh_TW/menu-ui-handler.ts [hunk 4]
```diff
@@ -18,6 +18,8 @@ export const menuUiHandler: SimpleTranslationEntries = {
   "exportSlotSelect": "選擇要導出的存檔位。",
   "importData": "導入數據",
   "exportData": "導出數據",
+  "importSettings": "Import Settings",
+  "exportSettings": "Export Settings",
```
Comment (RimKnight) [inline src/locales/zh_TW/menu-ui-handler.ts:22]:
```suggestion
  "importSettings": "導入設置",
  "exportSettings": "導出設置",
```

File: src/locales/ko/menu-ui-handler.ts [hunk 5]
```diff
@@ -18,6 +18,8 @@ export const menuUiHandler: SimpleTranslationEntries = {
   "exportSlotSelect": "내보낼 슬롯을 골라주세요.",
   "importData": "데이터 불러오기",
   "exportData": "데이터 내보내기",
+  "importSettings": "Import Settings",
+  "exportSettings": "Export Settings",
```
Comment (sodaMelon) [inline src/locales/ko/menu-ui-handler.ts:22]:
```suggestion
  "importSettings": "설정 불러오기",
  "exportSettings": "설정 내보내기",
```

**Review by sodaMelon — COMMENTED** [review]
good morning
i added Korean localization!

**Review by sodaMelon — DISMISSED** [review]

**Review by RimKnight — APPROVED** [review]

**Comment by NicusPulcis** [comment]
hey @prateau i've added italian

**Comment by prateau** [comment]
Fixed conflicts

**Comment by prateau** [comment]
> hey @prateau i've added italian

Hey I'm sorry but I don't see your change anywhere

**Review by josericardo-fo — DISMISSED** [review]

**Comment by NicusPulcis** [comment]
@prateau if you wanna add them yourself there they are:

"Import Settings" -> "Importa impostazioni"
"Export Settings" -> "Esporta impostazioni"

**Comment by prateau** [comment]
Conflict resolved

**Comment by flx-sta** [comment]
@prateau   closed for being stale (no updates for over a month)

----

Feel free to create a new PR or ask us to re-open this one.

**Pull request closed without merging**